src/components/SummaryKeyComponent.js

```diff
@@ -16,6 +18,7 @@ class SummaryKeyComponent {
    * @constructor
    */
   constructor() {
+    this.abstractComponent = new SemScholarAbstractComponent();
```

Comment:
It might be better to call it SemScholarInfo component or SemScholarAbstractAndTldr component.

Reply:
Done